alternative-checkout/README.md

```diff
@@ -0,0 +1,47 @@
+# Alternative checkout
+
+This example shows a way to alter the checkout flow by "merging" the address and shipping method steps from the checkout, while reusing most of Front-Commerce components.
+
+This example shows different practical use of [Configuration Providers](https://developers.front-commerce.com/docs/2.x/advanced/server/configurations#what-is-a-configuration-provider) in Front-Commerce.
```

Comment:
**issue:** seems unrelated (copy/paste?)

**suggestion:** remove
```suggestion
```

Reply:
Indeed ! I pasted it to have an example under my eye and forgot to remove it 🤦